panic: runtime error: index out of range [0] with length 0

## Expected behavior
Display user friendly info.

## Actual behavior
```
[10:58:35] xcdr:~ $ mc admin info server minio-3s
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/minio/mc/cmd.infoMessage.String(0x0, 0x0, 0xd25729, 0x2, 0xc0002f4220, 0x14, 0x0, 0x0, 0xc0002172c0, 0x2, ...)
        /q/.q/sources/gopath/src/github.com/minio/mc/cmd/admin-info-server.go:176 +0x21af
github.com/minio/mc/cmd.printMsg(0xe61560, 0xc0002454a0)
        /q/.q/sources/gopath/src/github.com/minio/mc/cmd/print.go:30 +0xeb
github.com/minio/mc/cmd.mainAdminServerInfo(0xc0002ae680, 0x0, 0x2ae680)
        /q/.q/sources/gopath/src/github.com/minio/mc/cmd/admin-info-server.go:353 +0x80c
github.com/minio/cli.HandleAction(0xc0ad20, 0xd7ab78, 0xc0002ae680, 0xc0000d7b00, 0x0)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/app.go:498 +0xc8
github.com/minio/cli.Command.Run(0xd29172, 0x6, 0x0, 0x0, 0x0, 0x0, 0x0, 0xd43304, 0x20, 0x0, ...)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/command.go:225 +0x991
github.com/minio/cli.(*App).RunAsSubcommand(0xc0002ae000, 0xc00025bd40, 0x0, 0x0)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/app.go:387 +0x7fa
github.com/minio/cli.Command.startApp(0xd26d72, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0xd43304, 0x20, 0x0, ...)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/command.go:313 +0x809
github.com/minio/cli.Command.Run(0xd26d72, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0xd43304, 0x20, 0x0, ...)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/command.go:111 +0x127b
github.com/minio/cli.(*App).RunAsSubcommand(0xc00025ba00, 0xc00025a4e0, 0x0, 0x0)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/app.go:387 +0x7fa
github.com/minio/cli.Command.startApp(0xd2785a, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0xd35d86, 0x14, 0x0, ...)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/command.go:313 +0x809
github.com/minio/cli.Command.Run(0xd2785a, 0x5, 0x0, 0x0, 0x0, 0x0, 0x0, 0xd35d86, 0x14, 0x0, ...)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/command.go:111 +0x127b
github.com/minio/cli.(*App).Run(0xc00025a1a0, 0xc0000ba000, 0x5, 0x5, 0x0, 0x0)
        /q/.q/sources/gopath/pkg/mod/github.com/minio/cli@v1.22.0/app.go:261 +0x6ab
github.com/minio/mc/cmd.Main(0xc0000ba000, 0x5, 0x5)
        /q/.q/sources/gopath/src/github.com/minio/mc/cmd/main.go:110 +0x29c
main.main()
        /q/.q/sources/gopath/src/github.com/minio/mc/main.go:34 +0x45
```
## Steps to reproduce the behavior
* create 3 minio instances [minio-1,minio-2,minio-3]
* setup minio client for minio-1 instance and name it minio-3s (for this example)
* run `mc admin info server minio-3s`
* stop minio-2 and minio-3 instances
* run again `mc admin info server minio-3s`

## mc version
```
Version: 2019-10-09T22:54:57Z
Release-tag: RELEASE.2019-10-09T22-54-57Z
Commit-id: f93fe1330a3647b1afaff0ed8c188d2897bf391e
```
## System information
Minio instances:
```
root@minio-1:~ # freebsd-version
12.0-RELEASE-p11

root@minio-1:~ # minio version
Version: 2019-08-29T00:25:01Z
Release-Tag: RELEASE.2019-08-29T00-25-01Z
Commit-ID: eb18c82976251d9a0c0730df27e10204c89beae2
```

Both minio clients on FreeBSD and Ubuntu (WSL) have the same issue.

--- a/cmd/admin-info-server.go
+++ b/cmd/admin-info-server.go
@@ -180,7 +180,7 @@ func (u infoMessage) String() (msg string) {
 		downBackends := 0
 		for _, set := range v.Sets {
 			for _, s := range set {
-				if strings.Contains(s.Endpoint, u.Addr) || s.Endpoint[0] == '/' || s.Endpoint[0] == '.' {
+				if len(s.Endpoint) > 0 && (strings.Contains(s.Endpoint, u.Addr) || s.Endpoint[0] == '/' || s.Endpoint[0] == '.') {
 					if s.State == "ok" {
 						upBackends++
 					} else {

